app/views/payments/cybersource_form.erb

```diff
@@ -0,0 +1,15 @@
+<html charset="utf-8">
```

Comment:
Shouldn't this filename end with `.html.erb`

Reply:
oh yeah, good catch!

Reply:
fixed